health-integrations/src/main/java/com/netflix/runtime/health/eureka/EurekaHealthStatusBridgeModule.java

```diff
@@ -33,38 +33,48 @@
 import com.netflix.governator.spi.LifecycleListener;
 import com.netflix.runtime.health.api.HealthCheckAggregator;
 import com.netflix.runtime.health.api.HealthCheckStatus;
+import com.netflix.runtime.health.api.IndicatorFilter;
 import com.netflix.runtime.health.core.HealthCheckStatusChangedEvent;
+import com.netflix.runtime.health.status.ArchaiusHealthStatusFilterModule;
 
 /**
- * Installing this module couples Eureka status (UP/DOWN/STARTING) to {@link HealthCheckStatus}. 
- * After injector creation, Eureka will be provided with a {@link HealthCheckHandler} instance 
- * which delegates to {@link HealthCheckAggregator}. A "healthy" status will report UP and "unhealthy" 
- * will report DOWN in Eureka.
+ * Installing this module couples Eureka status (UP/DOWN/STARTING) to {@link HealthCheckStatus}. After injector creation, Eureka will be provided
+ * with a {@link HealthCheckHandler} instance which delegates to
+ * {@link HealthCheckAggregator}. A "healthy" status will report UP and
+ * "unhealthy" will report DOWN in Eureka.
  * 
- * Please note that prior to injector creation being completed, Eureka will remain at its default status
- * of STARTING unless it is explicitly set otherwise. 
+ * Please note that prior to injector creation being completed, Eureka will
+ * remain at its default status of STARTING unless it is explicitly set
+ * otherwise.
  */
 public class EurekaHealthStatusBridgeModule extends AbstractModule {
 
     @Override
     protected void configure() {
-       bind(ApplicationEventListener.class).to(EurekaHealthStatusInformingApplicationEventListener.class).asEagerSingleton();
+        bind(ApplicationEventListener.class).to(EurekaHealthStatusInformingApplicationEventListener.class).asEagerSingleton();
     }
 
     private static class EurekaHealthStatusInformingApplicationEventListener
             implements ApplicationEventListener<HealthCheckStatusChangedEvent>, LifecycleListener {
 
-        @com.google.inject.Inject(optional=true)
+        @com.google.inject.Inject(optional = true)
         private ApplicationEventDispatcher eventDispatcher;
-        @Inject private ApplicationInfoManager applicationInfoManager;
-        @Inject private HealthCheckAggregator healthCheckAggregator;
-        @Inject private EurekaClient eurekaClient;
+        @Inject
+        private ApplicationInfoManager applicationInfoManager;
```

Comment:
I'm stuck doing field injection because of the optional inject. Not aware of any other way to get around this with Guice.